build.xml

```diff
@@ -124,6 +124,9 @@
     <delete>
       <fileset dir="test/python/scripts/spl_app" includes="toolkit.xml"/>
     </delete>
+    <delete>
+      <fileset dir="test/python/scripts/sc_test_files" includes="test_sc_old_toolkit_name.txt"/>
```

Comment:
Not too familiar with how the ant build process works, does this delete the directory `scripts/sc_test_files`, or just the file `test_sc_old_toolkit_name.txt` ?

Reply:
just the file